src/test/java/org/ssu/edu/teachua/utils/runners/BaseTestRunnerUI.java

```diff
@@ -1,24 +1,37 @@
-package org.ssu.edu.teachua.utils.runners;
+package org.ssu.edu.teachua.ui.runners;
 
 import io.github.bonigarcia.wdm.WebDriverManager;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.chrome.ChromeOptions;
-import org.ssu.edu.teachua.utils.TestNgListeners;
+import org.openqa.selenium.support.FindBy;
 import org.ssu.edu.teachua.utils.TestValueProvider;
+import org.ssu.edu.teachua.utils.runners.Browsers;
 import org.testng.ITestContext;
 import org.testng.annotations.*;
 import org.testng.asserts.SoftAssert;
 
 import java.time.Duration;
+import java.util.Arrays;
+import java.util.List;
 
-@Listeners(TestNgListeners.class)
 public class BaseTestRunnerUI {
 
     protected static TestValueProvider valueProvider;
     protected WebDriver driver;
     protected SoftAssert softAssert = new SoftAssert();
+    Browsers browsers = new Browsers();
+
+    private void checkErrorPage(String browser) throws InterruptedException {
+        try {
```

Comment:
Do we need throw an InterruptedException ?

Reply:
fix